packages/get-packages/src/index.ts

```diff
@@ -6,21 +6,19 @@ import fs from "fs-extra";
 import path from "path";
 import globby, { sync as globbySync } from "globby";
 import readYamlFile, { sync as readYamlFileSync } from "read-yaml-file";
-import { PackageJSON } from "@changesets/types";
 import { findRoot, findRootSync } from "@manypkg/find-root";
-
-const isNoEntryError = (err: unknown): boolean =>
-  !!err && typeof err === "object" && "code" in err && err.code === "ENOENT";
-
-export type Tool = "yarn" | "bolt" | "pnpm" | "lerna" | "root";
-
-export type Package = { packageJson: PackageJSON; dir: string };
-
-export type Packages = {
-  tool: Tool;
-  packages: Package[];
-  root: Package;
-};
+import {
+  Tool,
+  ToolType,
+  Package,
+  Packages,
+  MonorepoRoot,
+  PackageJSON,
+  supportedTools,
+  defaultOrder,
+} from "@manypkg/tools";
+
+export { Package, Packages } from "@manypkg/tools";
```

Comment:
Since we are using `isolatedModules` this is required here:
```suggestion
export type { Package, Packages } from "@manypkg/tools";
```